client/src/App.js

```diff
@@ -24,47 +25,49 @@ import './App.css';
 function App() {
 	return (
 		<div className='App'>
-			<ScrollToAnchor />
-			<Header />
-			<Routes>
-				<Route index element={<Navigate to='/home' />} />
-				<Route path='/about' element={<About />} />
-				<Route path='/contactus' element={<Contactus />} />
-				<Route path='/docs' element={<ApiDocs />} />
-				<Route path='/forgot-password' element={<ForgotPassword />} />
-				<Route path='/pricing' element={<Pricing />} />
-				<Route path='/reset-password' element={<ResetPassword />} />
-				<Route path='/signin' element={<Signin />} />
-				<Route path='/signup' element={<Signup />} />
-				<Route path='/home' element={<Home />} />
-				<Route path='/verify' element={<Verification />} />
-				<Route
-					path='/profile'
-					element={
-						<ProtectedRoute>
-							<Account />
-						</ProtectedRoute>
-					}
-				/>
-				<Route
-					path='/admin'
-					element={
-						<ProtectedRoute adminOnly={true}>
-							<AdminDashboard />
-						</ProtectedRoute>
-					}
-				/>
-				<Route
-					path='/dashboard'
-					element={
-						<ProtectedRoute>
-							<Dashboard />
-						</ProtectedRoute>
-					}
-				/>
-				<Route path='*' element={<NotFound />} />
-			</Routes>
-			<Footer />
+			<ScrollProvider>
```

Comment:
Move <ScrollProvider>  component to `index.js` file. Keep all context provider at one place.

Reply:
> Move component to `index.js` file. Keep all context provider at one place.

@Asin-Junior-Honore